docs/api-reference/event-manager.md

```diff
@@ -59,23 +59,27 @@ Set the DOM element on which event handlers will be registered. If element has b
 
 Register an event handler function to be called on `event`.
 
-`eventManager.on(event, handler, srcElement)`
+`eventManager.on(event, handler, options)`
 
 - `event` {string|Object} - An event name (`String`) or map of event names to handlers.
 - `[handler]` {Function} - The function to be called on `event`.
-- `[srcElement]` {Node} - The source element of this event. If provided, only events that are targeting this element or its decendants will invoke the handler. If ignored, default to the whole document. Events are propagated up the DOM tree.
+- `[options]` {Object}
+  - `[srcElement]` {Node} - The source element of this event. If provided, only events that are targeting this element or its decendants will invoke the handler. If ignored, default to the root element of the event manager. Events are propagated up the DOM tree.
+  - `[priority]` {Number} - Hanlders targeting the same `srcElement` will be executed by their priorities (higher priorities first). Hanlders with the same priority will be executed in the order of registration.
```

Comment:
I would be a bit more explicit. Does "higher priority" mean a lower number or a higher number?